creduce/get_key.py

```diff
@@ -0,0 +1,40 @@
+import time
+import sys
+import sys
+import tty
+import termios
+import select
+
+def get_key():
+
+    (ready, _, _) = select.select([fd], [], [], 0)
+
+    if ready:
+        key = sys.stdin.read(1)
+    else:
+        key = None
+
+    return key
+
+if __name__ == "__main__":
+    fd = sys.stdin.fileno()
+    old_settings = termios.tcgetattr(fd)
+    tty.setraw(sys.stdin.fileno())
+
+    i = 0
+
+    while True:
```

Comment:
Opps, that file was not supposed to be committed. I used it to experiment with the "read key to abort pass" feature. If I remember correctly I have not found a solution for that yet. At least not one that works for Linux and Windows.

Is that feature an absolute must have?